osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SliderBall.cs

```diff
@@ -97,6 +98,12 @@ protected override bool OnMouseMove(InputState state)
             return base.OnMouseMove(state);
         }
 
+        public override bool ReceiveMouseInputAt(Vector2 screenSpacePos)
+        {
+            // If the current time is between the start and end of the slider, we should track mouse input regardless of the cursor position.
+            return canCurrentlyTrack || base.ReceiveMouseInputAt(screenSpacePos);
+        }
+
         private bool tracking;
         public bool Tracking

```

Comment:
i think this should be a private set, since we seem to be calculating trackability privately in all cases.